### PR TITLE
Document using a custom CoreDNS image repository

### PR DESCRIPTION
**Pull request by praseodym**
Fixes kubernetes/kubeadm#821.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/coredns.md [hunk 1]
```diff
@@ -31,6 +31,15 @@ kubeadm init --feature-gates=CoreDNS=true
 
 This installs CoreDNS instead of kube-dns.
 
+## Using a custom CoreDNS image repository with kubeadm
+
+To use a custom image repository for the CoreDNS image, e.g. located in your own Docker registry,
```
[the remark this answers is not in the document]
Comment (praseodym) [reply content/en/docs/tasks/administer-cluster/coredns.md]: Thank you, updated!

**Comment by neolit123** [comment]
@tengqm 

this one is for `1.10` as it's a generic CoreDNS change, since CoreDNS is already available to the users.

however, we are switching to CoreDNS **by default** and all changes related to that should go against `release-1.11`. i'm actually about to start working on a PR for it and will attempt to cherry-pick/close other PRs and keep the work centralized...

**Comment by johnbelamaric** [comment]
thanks!

/lgtm

**Pull request merged**